docs/source/install/overview.rst

```diff
@@ -0,0 +1,62 @@
+Overview: Single-box Reference Deployment
+=========================================
+
+.. todo:: short intro paragraph: this is reference deployment for single box, follow installation
+    guide to get here.
+
+.. figure:: https://docs.google.com/drawings/d/1X6u8BB9bnWkW8C81ERBvjIKRfo9mDos4XEKeDv6YiF0/pub?w=960&amp;h=720
```

Comment:
Thanks for the comments. I'll edit the pic but KEEP it as complex as it is. This is not a "first" picture folks see; the simpler one is in the overview. Somewhere ( and I assume, here) we need a good wiring diagram for someone who wants to understand how things are really related.